Description: Fix some typos in c and perl code
 Fix typos in Slurm.pm common/plugstack.c sacctmgr/account_functions.c
 sacctmgr/user_functions.c causing spelling-error-in-binary in several
 binaries
Origin: maintainer
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=2921
Last-Update: <2016-07-19>

--- slurm-llnl-16.05.2.orig/contribs/perlapi/libslurm/perl/lib/Slurm.pm
+++ slurm-llnl-16.05.2/contribs/perlapi/libslurm/perl/lib/Slurm.pm
@@ -371,7 +371,7 @@ Allocate resources for a job request. If
 
 =over 2
 
-=item * IN $job_desc: description of resource allocation request, with sturcture of C<job_desc_msg_t>.
+=item * IN $job_desc: description of resource allocation request, with structure of C<job_desc_msg_t>.
 
 =item * RET: response to request, with structure of C<resource_allocation_response_msg_t>.  This only represents a job allocation if resources are immediately available.  Otherwise it just contains the job id of the enqueued job request. On failure C<undef> is returned.
     
@@ -383,7 +383,7 @@ Allocate resources for a job request.  T
     
 =over 2
 
-=item * IN $job_desc: description of resource allocation request, with sturcture of C<job_desc_msg_t>.
+=item * IN $job_desc: description of resource allocation request, with structure of C<job_desc_msg_t>.
 
 =item * IN $timeout: amount of time, in seconds, to wait for a response before giving up. A timeout of zero will wait indefinitely.    
 
--- slurm-llnl-16.05.2.orig/src/common/plugstack.c
+++ slurm-llnl-16.05.2/src/common/plugstack.c
@@ -1088,7 +1088,7 @@ static int _add_one_option(struct option
 
 	if (optz_add(optz, &opt) < 0) {
 		if (errno == EEXIST) {
-			error ("Ingoring conflicting option \"%s\" "
+			error ("Ignoring conflicting option \"%s\" "
 			       "in plugin \"%s\"",
 			       opt.name, spopt->plugin->name);
 		} else {
--- slurm-llnl-16.05.2.orig/src/sacctmgr/account_functions.c
+++ slurm-llnl-16.05.2/src/sacctmgr/account_functions.c
@@ -718,7 +718,7 @@ extern int sacctmgr_list_account(int arg
 	}
 
 	if (!acct_cond->with_assocs && cond_set > 1) {
-		if (!commit_check("You requested options that are only vaild "
+		if (!commit_check("You requested options that are only valid "
 				 "when querying with the withassoc option.\n"
 				 "Are you sure you want to continue?")) {
 			printf("Aborted\n");
--- slurm-llnl-16.05.2.orig/src/sacctmgr/user_functions.c
+++ slurm-llnl-16.05.2/src/sacctmgr/user_functions.c
@@ -1418,7 +1418,7 @@ extern int sacctmgr_list_user(int argc,
 	}
 
 	if (!user_cond->with_assocs && cond_set > 1) {
-		if (!commit_check("You requested options that are only vaild "
+		if (!commit_check("You requested options that are only valid "
 				 "when querying with the withassoc option.\n"
 				 "Are you sure you want to continue?")) {
 			printf("Aborted\n");
